core/sail/nativerdf/src/test/java/org/eclipse/rdf4j/sail/nativerdf/LongMultithreadedTransactions.java

```diff
@@ -73,6 +75,7 @@ public void test() {
 	}
 
 	@Test
+	@Ignore
```

Comment:
Why ignore these tests?

Reply:
I created this to diagnose the previous race condition with multithreaded access to the NativeStore when using SERIALIZABLE transactions. The test never completes, since the purpose is to just bash the NativeStore until it crashes. The class doesn't have "Test" in it, so maven doesn't pick it up. And I'm getting a bit annoyed at running it by mistake when running all the NativeStore tests from IntelliJ. Should probably add some docs to the tests to explain why.